analytics-core/build.gradle

```diff
@@ -0,0 +1,60 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+ext {
+	publishedGroupId = 'com.liferay'
+	libraryName = 'com.liferay.analytics.client.android'
+	libraryDescription = 'Liferay Analytics client for Android'
+
+	bintrayRepo = 'liferay-mobile'
+	bintrayOrg = 'liferay'
+
+	versionCode = 100
+	versionName = '1.0.0'
+
+	siteUrl = 'https://github.com/liferay-mobile/com-liferay-analytics-client-android'
+	gitUrl = 'https://github.com/liferay-mobile/com-liferay-analytics-client-android.git'
+
+	licenseName = 'GNU Lesser General Public License, version 2.1'
+	licenseUrl = 'http://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html'
+	allLicenses = ["LGPL-2.1"]
```

Comment:
Can you use simple ticks in gradle when you are not interpollating? If you have a $var+"" you have to use double quotes, but in the rest of the cases we should use single.